src/main/kotlin/community/flock/eco/workday/controllers/ExpenseController.kt

```diff
@@ -139,6 +139,7 @@ private fun CostExpense.produce() =
         date = date.toString(),
         status = status.produce(),
         expenseType = ExpenseType.COST,
+        travelDetails = null,
```

Comment:
FYI: wirespec had a breaking change in terms where default values (null) for nullable fields were removed